etc/kayobe/inventory/group_vars/compute/firewall

```diff
@@ -0,0 +1,88 @@
+---
+
+firewallgen_enable_firewall: True
+
+firewallgen_ipv4_input_allow_rules:
+  - interface: "lo"
+    port: 25
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'master'"
+  - interface: "lo"
+    port: 6633
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'neutron-openvsw' in docker container 'neutron_openvswitch_agent'"
+  - interface: "lo"
+    port: 6640
+    proto: tcp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'ovsdb-server' in docker container 'openvswitch_db'"
+  - interface: "{{ admin_oc_net_interface }}"
+    port: 22
+    proto: tcp
+    comment: "hint: used by 'sshd'"
+  - interface: "{{ internal_net_interface }}"
+    port: 8022
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'sshd' in docker container 'nova_ssh'"
+  - interface: "{{ internal_net_interface }}"
+    port: 9100
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'node_exporter' in docker container 'prometheus_node_exporter'"
+  - interface: "{{ internal_net_interface }}"
+    port: 16509
+    proto: tcp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'libvirtd' in docker container 'nova_libvirt'"
+  - interface: "all"
+    port: 123
+    proto: udp
+    comment: "hint: used by 'ntpd'"
+  - interface: "docker0"
+    port: 123
+    proto: udp
+    destination: "172.17.0.1"
+    comment: "hint: used by 'ntpd'"
+  - interface: "lo"
+    port: 123
+    proto: udp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'ntpd'"
+  - interface: "lo"
+    port: 323
+    proto: udp
+    destination: "127.0.0.1"
+    comment: "hint: used by 'chronyd' in docker container 'chrony'"
+  - interface: "{{ admin_oc_net_interface }}"
+    port: 123
+    proto: udp
+    destination: "{{ admin_oc_net_name | net_ip }}"
+    comment: "hint: used by 'ntpd'"
+  - interface: "{{ external_net_interface }}"
+    port: 123
+    proto: udp
+    destination: "{{ 'external_net' | net_ip }}"
+    comment: "hint: used by 'ntpd'"
+  - interface: "{{ internal_net_interface }}"
+    port: 123
+    proto: udp
+    destination: "{{ internal_net_name | net_ip }}"
+    comment: "hint: used by 'ntpd'"
```

Comment:
Feels like we need to allow only one network for ntpd, I would prefer if that was admin_oc_net I think? follow the ssh traffic.

Reply:
I've restricted this to the internal interface for now.

Reply:
ah, got you, let's go with that than.